app/src/test/java/com/kickstarter/viewmodels/PledgeFragmentViewModelTest.kt

```diff
@@ -83,13 +89,47 @@ class PledgeFragmentViewModelTest : KSRobolectricTestCase() {
         this.cards.assertValueCount(2)
     }
 
+    @Test
+    fun testContinueButtonIsGone() {
```

Comment:
I wonder if this test is still relevant, since the `testLoggingInUser` method already tests the outputs in both (logged in / logged out) scenarios.

Reply:
Here's what I am thinking; there are 3 scenarios:
1. The user opens the PledgeFragment logged in
2. The user opens the PledgeFragment logged out
3. The user opens the PledgeFragment logged out and then logs in

Reply:
Oh, I see what you mean. So could you rename the tests to reflect what you want to test?

Reply:
༼ ༎ຶ ෴ ༎ຶ༽  but they are
1. `testPaymentContainerIsGone`
2. `testContinueButtonIsGone`
3. `testLoggingInUser` - maybe this one could be `testContinueButtonAndPaymentContainerAfterLogin`?

Reply:
lol I know. What I'm saying is - If the test is called `testPaymentContainerIsGone`, we should preferably only test the `paymentContainerIsGone` output. The same would apply for the `testContinueButtonIsGone`.

Currently we are testing 2 outputs (`paymentContainerIsGone` and `continueButtonIsGone`) on the same test . This is fine, but in this case, we should rename the tests to explicitly say we are testing a flow that involves both outputs and not only one at a time. (But this is just a suggestion.)